build-tools/scripts/RunNUnitTests.targets

```diff
@@ -5,7 +5,7 @@
     <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
     <_Runtime Condition=" '$(RUNTIME)' != '' ">$(RUNTIME)</_Runtime>
     <_Runtime Condition=" '$(RUNTIME)' == '' And '$(OS)' != 'Windows_NT' ">mono --debug</_Runtime>
-    <_NUnit>$(_Runtime) packages\NUnit.ConsoleRunner.3.9.0\tools\nunit3-console.exe</_NUnit>
+    <_NUnit>$(_Runtime) packages\nunit.consolerunner\3.9.0\tools\nunit3-console.exe</_NUnit>
```

Comment:
`@(PackageReference)` lowercases all the things?  Interesting...

Reply:
And adds the slash...